tools/virsh.c

```diff
@@ -510,6 +510,9 @@ virshShowVersion(vshControl *ctl G_GNUC_UNUSED)
 #ifdef WITH_OPENVZ
     vshPrint(ctl, " OpenVZ");
 #endif
+#ifdef WITH_CH
+    vshPrint(ctl, " Cloud Hypervisor");
```

Comment:
```suggestion
    vshPrint(ctl, " Cloud-Hypervisor");
```
Reason being is that if we use virsh to check supported Hypervisors, it will show:
```
$ virsh -V
Virsh command line tool of libvirt 6.7.0
See web site at https://libvirt.org/

Compiled with support for:
 Hypervisors: QEMU/KVM LXC OpenVZ Cloud Hypervisor VMware VirtualBox ESX Test
...
```

Notice that "Cloud Hypervisor" doesn't follow the syntax rule here as others (separated name by space).
@sameo I'd think making all apearance to be consistent with "Cloud-Hypervisor" is better.

Reply:
> Reason being is that if we use virsh to check supported Hypervisors, it will show:
> 
> ```
> $ virsh -V
> Virsh command line tool of libvirt 6.7.0
> See web site at https://libvirt.org/
> 
> Compiled with support for:
>  Hypervisors: QEMU/KVM LXC OpenVZ Cloud Hypervisor VMware VirtualBox ESX Test
> ...
> ```
> 
> Notice that "Cloud Hypervisor" doesn't follow the syntax rule here as others (separated name by space).
> @sameo I'd think making all apearance to be consistent with "Cloud-Hypervisor" is better.

I agree, yes.